aps/rms_jobs/copy_rms_param_to_fmu_grid.py

```diff
@@ -23,19 +25,15 @@ def run(project, **kwargs):
     """
     if project.current_realisation > 0:
         raise ValueError(
-            f'In RMS models to be used with a FMU loop in ERT,'
+            'In RMS models to be used with a FMU loop in ERT,'
             'the grid and parameters should be shared and realisation = 1'
         )
     params = {
         'project': project,
         'model_file_name': get_specification_file(
             _type=SpecificationType.RESAMPLE, _format=ModelFileFormat.YML, **kwargs
         ),
-        'debug_level': Debug.VERBOSE,
+        'debug_level': 2,
```

Comment:
We should probably still use the Enum so that it is clearer what the intension is

Reply:
OK, This script is used untill we remove the stubs.